css/styles.css

```diff
@@ -0,0 +1,177 @@
+/*----- Global Styles --------*/
+
+html {
+	height: 100%;
+}
+
+body {
+	font-family: 'Cookie', cursive;
+	height: 100%;
+}
+
```

Comment:
Determine a convention on spacing between sections. Some of the comments have 2 spaces above, somw have 2 below, some have 1 and 1.